cluster/vagrant/provision-master.sh

```diff
@@ -253,6 +253,19 @@ pushd /kube-install
   ./kubernetes/saltbase/install.sh "${server_binary_tar##*/}"
 popd
 
+# Enable Fedora Cockpit on host to support Kubernetes administration
+# Access it by going to <master-ip>:9090 and login as vagrant/vagrant
+if ! which /usr/libexec/cockpit-ws &>/dev/null; then
+  
+  pushd /etc/yum.repos.d
+    wget https://copr.fedoraproject.org/coprs/sgallagh/cockpit-preview/repo/fedora-21/sgallagh-cockpit-preview-fedora-21.repo
+    yum install -y cockpit cockpit-kubernetes  
+  popd
+
+  systemctl enable cockpit
```

Comment:
For bonus points should be:

 ```
systemctl enable cockpit.socket
systemctl start cockpit.socket
```

Cockpit is zero footprint, socket activated on demand. It doesn't need to be running all the time.